selinux/defaults/main.yml

```diff
@@ -0,0 +1 @@
+selinux_state: disabled
```

Comment:
is it really necessary to disable SELinux? - I would not like the security of my servers being reduced for no good reason.